parl/env/continuous_wrappers.py

```diff
@@ -26,6 +27,11 @@ def __init__(self, env):
         self.low_bound = self.env.action_space.low[0]
         self.high_bound = self.env.action_space.high[0]
         assert self.high_bound > self.low_bound
+        self._max_episode_steps = int(self.env._max_episode_steps)
+        try:
+            self._elapsed_steps = int(self.env._elapsed_steps)
+        except:
+            logger.error('object has no attribute _elspaed_steps')
```

Comment:
Don't display this error when user does not need to use env._elapsed_steps

Reply:
done